### DISCUSSION TITLE
Welcome OpenSauced to the Student Developer Pack!

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#114413] [GitHub Education]
![education-octocat-opensauded](https://github.com/community/community/assets/107881423/a6a3f35f-d0d5-4e51-a6e7-21bc8fc9210e)

### Ever wonder how to actually get started with open source? Now’s your chance! Let OpenSauced help you start your journey.

We're happy to share that OpenSauced is now a part of the Student Developer Pack! OpenSauced is a platform that helps you track and showcase your open source contributions, making it easier to get noticed by potential employers and collaborators.

As a special benefit to GitHub Education Learners, [OpenSauced](https://github.com/open-sauced) is offering free Workspaces, which means students can track repositories, contributors and maintainers, and their own work in one space. You can think of your profile as your current open source resume. You'll also have access to our latest courses.

To get started, [set up your OpenSauced account](https://oss.fyi/start-opensauced), and then take a look at the [OpenSauced Student Guide](https://oss.fyi/students-guide-GH) with all of the resources you need to get started. Want to really dive into the world of open source? You can do so by starting their [Intro to Open Source course](https://oss.fyi/intro-oss-course-GH) with tips, tools and great ways to make your first contributions or build on your existing portfolio.

Learn more about OpenSauced and their GitHub Student Developer Pack offer [here](https://opensauced.pizza/students).

More links & resources!

- [Watch the intro to Open Sauced on YouTube](https://www.youtube.com/watch?v=CKbTdYZAvSM&list=PLHyZ0Wz_A44XSYlBAfO2nBqoYaJJ5fdu5)
- [opensauced.pizza](https://opensauced.pizza/)
- [insights.opensauced.pizza](https://insights.opensauced.pizza/)
- [hot.opensauced.pizza](https://hot.opensauced.pizza/)
- [docs.opensauced.pizza](https://docs.opensauced.pizza/)
- [api.opensauced.pizza](https://api.opensauced.pizza/)

**Comment by pizofreude** [comment]
I wish GitHub would include Open Source Contributor Developer Pack as well. Fingers-crossed.

**Reply by finandolopes** [reply]
Sim

**Reply by victoriahello** [reply]
What does open sourced mean?

**Reply by pizofreude** [reply]
@victoriahello in essence it basically means anything that is publicly accessible, modifiable, and redistributable. Of course, the specifics depend on its adopted open source [licenses](https://opensource.org/licenses). Public domain uses [CC0 license](https://creativecommons.org/public-domain/) and my favorite for the majority of my open-source projects is [Apache 2.0 license](https://opensource.org/license/apache-2-0).

Feel free to read more about it [here](https://opensource.com/resources/what-open-source).

**Comment by lalithaar** [comment]
Hii I am wondering if this is no longer available because when I go https://app.opensauced.pizza/ I get redirected to this blog post https://opensauced.pizza/blog/opensauced-is-joining-the-linux-foundation
"OpenSauced is joining the Linux Foundation"

How can I access the resources, if it's indeed available? Thank you so much

**Reply by adawright** [reply]
Hi @lalithaar!

Thank you for reaching out!  Third party offers in the Student Developer Pack are subject to change at the request of the partner offering the benefit. When possible, the GitHub Education team will try to notify users in advance, but in this case service was disrupted. The good news is, that although OpenSauced was acquired and some of the resources previously in the SDP are no longer available, many of their offerings are still free and open source. You can still find many of their courses and videos available on [their GitHub](https://github.com/open-sauced) or through their ["Becoming a Maintainer" and "Intro to Open Source" courses](https://opensauced.pizza/learn/). 

We have also updated the offer to reflect this change in resources. To stay up-to-date on product offers, please visit [this public repository](https://github.com/github-education-resources/Student-Developer-Pack-Current-Partners-FAQ).

Hope this helps and please let us know if you have further questions!
